src/platform/with_wgl/native_gl_context.rs

```diff
@@ -0,0 +1,144 @@
+use platform::NativeGLContextMethods;
+use std::ffi::CString;
+use std::os::raw::{c_void};
+use std::ptr;
+use std::sync::{Once, ONCE_INIT};
+
+use winapi;
+use user32;
+use kernel32;
+use super::wgl;
+use super::wgl_attributes::*;
+use gleam::gl;
+
+/// Wrapper to satisfy `Sync`.
+struct HMODULEWrapper(winapi::HMODULE);
+unsafe impl Sync for HMODULEWrapper {}
+
+lazy_static! {
+    static ref GL_LIB: Option<HMODULEWrapper>  = {
+        let p = unsafe{kernel32::LoadLibraryA(b"opengl32.dll\0".as_ptr() as *const _)};
+        if p.is_null() {
+            debug!("opengl32.dll not found!");
+            None
+        }
+        else {
+            debug!("opengl32.dll LOADED!");
+            Some(HMODULEWrapper(p))
+        }
+    };
+}
+
+static LOAD_GL: Once = ONCE_INIT;
+pub fn load_gl() {
```

Comment:
I thought about that too but we might need an API change to avoid loading the symbols under the hood in WGL. The callers can't load the symbols because wglGetProcAddress only works in the presence of a valid GL context. On the other hand If a users creates a context before loading symbols,  we call some gl:: functions to create the FBO, texture, etc, and we need symbols loaded for that. 

Some possible solutions I'm not in love with, but will work:

* Create a dummy context in NativeGLContext::getProcAdress if there isn't one bound
* Separate the context creation and the FBO initialization. The caller would have to load the symbols after the context creation and before the FBO initialization.